Libraries/Components/View/ViewPropTypes.windows.js

```diff
@@ -87,6 +87,8 @@ export type ViewProps = {
 module.exports = {
   ...TVViewPropTypes,
 
+  allowDrop: PropTypes.bool,
```

Comment:
Add a comment here.

Reply:
Also, I don't think these are necessarily alphabetical. Can you move this to the bottom with `manipulationModes` so all the windows-specific props are clustered.

---
In reply to: [159500497](https://github.com/Microsoft/react-native-windows/pull/1549#discussion_r159500497) [](ancestors = 159500497)

Reply:
Fixed